variables.tf

```diff
@@ -207,7 +209,8 @@ variable "administrative_stack_drift_detection_reconcile" {
 variable "administrative_stack_drift_detection_schedule" {
   type        = list(string)
   description = "List of cron expressions to schedule drift detection for the administrative stack"
-  default     = ["0 */24 * * *"]
+  default = [
+  "0 */24 * * *"]
```

Comment:
nit: this is weird but super low priority.

Reply:
fixed